DESCRIPTION.md

```diff
@@ -19,6 +19,7 @@ Source code is also available at:
   - Add support for iceberg table with Snowflake Catalog
   - Fix cluster by option to support explicit expressions
   - Add support for MAP datatype
+  - Add support for partition by to copy into <location>
```

Comment:
Add this to Unreleased section

Reply:
Thanks. Changed in https://github.com/snowflakedb/snowflake-sqlalchemy/pull/542/commits/300a4fb672984309698c059840e6aaf24193268a